vendor/code.gitea.io/git/repo.go

```diff
@@ -153,6 +153,13 @@ func Push(repoPath, remote, branch string) error {
 	return err
 }
 
+// PushFroce pushs local commits to given remote branch and forces any changes.
+func PushForce(repoPath, remote, branch string) error {
+	_, err := NewCommand("push", "--force", "--all", remote, branch).RunInDir(repoPath)
+	_, err = NewCommand("push", "--force", "--tags", remote, branch).RunInDir(repoPath)
```

Comment:
Should there be a `if err != nil { return err }` check before this line? It seems that we completely ignore the `err` value returned by the first call.